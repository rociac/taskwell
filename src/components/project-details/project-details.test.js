import ProjectDetails from './project-details.component';
import renderer from 'react-test-renderer';
import { cleanup } from '@testing-library/react';

afterEach(cleanup);

describe('Project Details', () => {
  it('renders properly with props', () => {
    const mockProps = {
      name: 'Test Project',
      type: 'Software',
      description: 'Test description',
      liveLink: 'www.example.com',
      image: 'www.example.com/image'
    };
    const tree = renderer.create(<ProjectDetails {...mockProps}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});