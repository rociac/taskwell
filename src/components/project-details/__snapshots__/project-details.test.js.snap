// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Details renders properly with props 1`] = `
<div
  className="container"
>
  <div
    className="imageContainer"
  >
    <img
      alt=""
      src="www.example.com/image"
    />
  </div>
  <div
    className="textContainer"
  >
    <div
      className="header"
    >
      <p
        className="name"
      >
        Test Project
      </p>
      <p
        className="type"
      >
        Software
      </p>
    </div>
    <p
      className="description"
    >
      Test description
    </p>
    <p
      className="link"
    >
      Visit the project 
      <a
        href="www.example.com"
      >
        here
      </a>
    </p>
  </div>
</div>
`;
